src/Tmds.ExecFunction/ExecFunction.cs

```diff
@@ -163,12 +180,16 @@ private static void ConfigureProcessStartInfoForMethodInvocation(MethodInfo meth
             // If we need the host (if it exists), use it, otherwise target the console app directly.
             Type t = method.DeclaringType;
             Assembly a = t.GetTypeInfo().Assembly;
-            string programArgs = PasteArguments.Paste(new string[] { a.FullName, t.FullName, method.Name });
+            string programArgs = PasteArguments.Paste(new string[] { a.FullName, t.FullName, method.Name, debuggerEnabled.ToString() });
             string functionArgs = PasteArguments.Paste(args);
-            string fullArgs = HostArguments + " " + " " + programArgs + " " + functionArgs;
+            string fullArgs = execFunctionHostArgs.HostArguments + " " + " " + programArgs + " " + functionArgs;
 
-            psi.FileName = HostFilename;
+            psi.FileName = execFunctionHostArgs.HostFilename;
             psi.Arguments = fullArgs;
+            if(debuggerEnabled)
```

Comment:
S `RedirectStandardInput` after `configure?.Invoke(options);` so the user can't mess with it.